src/main/java/utilities/CartObject.java

```diff
@@ -0,0 +1,8 @@
+package utilities;
+
+public class CartObject {
+        public String Name;
+        public String Type;
+        public double Price;
+        public double Weight;
+}
```

Comment:
I think it's should be category instead of type

also keep about java convention the variables should be with lowercase:
name, type, price, weight